ga/htdocs/components/95_ga_configs.js

```diff
@@ -758,6 +758,31 @@ Ensembl.GA.eventConfigs.push(
                       },
     action          : function () { return this.getURL(); },
     label           : function() { return $(this.currentTarget).text(); }
+  },
+
+  // Track Run VEP button
+  {
+    id              : 'RunVEPButton',
+    event           : 'click',
+    wrapper         : '.ajax.initial_panel',
+    selector        : 'input.run_button',
+    category        : 'RunVEPButton',
+    action          : 'Tools/VEP',
+    label           : ''
```

Comment:
i would advise to put something in the labels for both, in google analytics you can view events by label and we occasionally filter by label when viewing reports in google analytics. Otherwise it will be blank for these. Maybe something like 'VEP Instant Button' and 'Run VEP Button' will do.

Reply:
Also change category to 'Tools' for both. @ens-st3 what do you think about the name for the category, will 'tools' be good or is too generic?